prow/jobs/generated/knative-extensions/eventing-kafka-broker-main.gen.yaml

```diff
@@ -352,7 +352,7 @@ periodics:
       - name: PLATFORM
         value: linux/ppc64le
       - name: KO_DOCKER_REPO
-        value: docker-na.artifactory.swg-devops.com/sys-linux-power-team-ftp3distro-docker-images-docker-local/knative
```

Comment:
Could you explain why don't we use the gcr.io (or newer Artifact Registry) we use on upstream Knative in all other places?

Reply:
@cardil  We've transitioned to IBM Cloud Registry (ICR) primarily because using gcr.io would result in additional costs, which the management won't approve.

Reply:
Okay. I can approve this, as this doesn't change the "status quo" - we move from one out-of-control service to another.

In the long run, we need to migrate this to a place, that the upstream Knative community controls. Or, if we can't remove those jobs from the upstream project (unfavorable route).

This is very similar to https://github.com/knative/infra/issues/344#issuecomment-1944174280

Reply:
FYI: https://github.com/knative/infra/pull/371#discussion_r1504217436

/cc @dprotaso 
/cc @krsna-m 
/cc @upodroid

Reply:
@cardil Yes , that's the end goal . For time-being we're hosting all the ppc64le/s390x specific images to ICR . There's already efforts in making those images as discussed in #344